src/nft/components/bag/BagRow.tsx

```diff
@@ -91,7 +106,7 @@ export const BagRow = ({ asset, usdPrice, removeAsset, showRemove, grayscale, is
             {asset.collectionIsVerified && <VerifiedIcon className={styles.icon} />}
           </Row>
         </Column>
-        {cardHovered && showRemove && (
+        {cardHovered && showRemove && !isMobile && (
```

Comment:
nit: Could make this condition a const like `showRemoveButton` and then the check for showing the price could just be `!showRemoveButton`